src/main/java/org/fenixedu/learning/domain/degree/DegreeSite.java

```diff
@@ -72,4 +90,10 @@ public void delete() {
         return Stream.of(categoryForSlug("announcement")).filter(Objects::nonNull).map(Category::makeWrap);
     }
 
+    private CMSFolder folderForPath(MenuContainer parent, String path) {
```

Comment:
Ok, like ExecutionCourseSite example, if parent folder not found, now is created a new one.